### DISCUSSION TITLE
SingleTaskMultiFidelityGP model

### DISCUSSION THREAD

**Discussion by S3anaaaaz** [meta-pytorch/botorch#1284] [General]
Hi,
Why in SingleTaskMultiFidelityGP you are using 4 different Materns with different priors (Gamma (3,6) and Gamma (6,2))?
What is the point of summing up all those kernels?
I also figured out that I cannot fit a good model with his surrogate on even for example Rosenbrock function (Multi-fidelity version) compared to other GPs.  attached a figure of that. Why does it train very bad?
![image](https://user-images.githubusercontent.com/104162097/176981013-baea32aa-bb83-4fce-916c-b3f08f0fd65b.png)

**Comment by Balandat** [comment]
> Why in SingleTaskMultiFidelityGP you are using 4 different Materns 

See here: https://github.com/pytorch/botorch/blob/main/botorch/models/gp_regression_fidelity.py#L41-L45

> Why in SingleTaskMultiFidelityGP you are using 4 different Materns 

You'll need to provide a bit more information here - what are you fitting the model to and what are you predicting? Can you please attach the code to repro?

**Comment by S3anaaaaz** [comment]
Thank you for your answer. I'm using the Wing example as an input and I
separated a part of the data for prediction.
The attached code is the comparison for 4 models : ExactGP, LMGP,
SingletaskGP and Single-task-multifidelity-GP.
As you see, ExactGP and LMGP work well, but the other ones cannot fit a
good model with that data, I should increase data to improve their
performance.
